docker/Dockerfile.build

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /usr/src
+
+RUN set -ex \
+	&& apt-get update \
+	&& apt-get -y dist-upgrade \
+	&& apt-get install -qq --no-install-recommends \
+	git ca-certificates \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/OrchidTechnologies/orchid.git
+
+WORKDIR orchid
+
+RUN git config --global url."git://git.savannah.gnu.org/".insteadOf "https://git.savannah.gnu.org/git/" \
+	&& git config --global url."git://git.savannah.nongnu.org/".insteadOf "https://git.savannah.nongnu.org/git/" \
+	&& git config --global advice.detachedHead false
+
+RUN git submodule update --init --recursive --jobs 3
```

Comment:
Nice idea and seems to work well for me.  It behaves a little weird on the mac version of docker since that lets you set how many cpus are available to the docker engine and does not default to all of them, but it still works there.  Change made.